apps/src/code-studio/pd/workshop_dashboard/workshop.jsx

```diff
@@ -166,6 +174,75 @@ export class Workshop extends React.Component {
     });
   };
 
+  handleMoveEnrollments = (destinationWorkshopId, selectedEnrollments) => {
+    const enrollmentIds = selectedEnrollments.map(enrollment => {
+      return enrollment.id;
+    });
+    const urlParams = `destination_workshop_id=${destinationWorkshopId}&enrollment_ids[]=${enrollmentIds.join(
+      '&enrollment_ids[]='
+    )}`;
+    this.moveEnrollmentRequest = $.ajax({
+      method: 'POST',
+      url: `/api/v1/pd/enrollments/move?${urlParams}`,
+      traditional: true
+    })
+      .done(() => {
+        // reload
+        this.loadEnrollments();
+        this.moveEnrollmentRequest = null;
+      })
+      .fail(() => {
+        this.setState({
+          error: 'Error: unable to move enrollments'
+        });
+        this.loadEnrollments();
+        this.moveEnrollmentRequest = null;
+      });
+  };
+
+  handleClickSelect = enrollment => {
+    if (
+      this.state.selectedEnrollments.findIndex(e => e.id === enrollment.id) >= 0
+    ) {
+      this.setState(state => {
+        const selectedEnrollments = state.selectedEnrollments.filter(e => {
+          return e.id !== enrollment.id;
+        });
+        return {selectedEnrollments};
+      });
+    } else {
+      this.setState(state => {
+        state.selectedEnrollments.push({
```

Comment:
Would it make sense for state to just hold the selected IDs, rather than duplicating this additional enrollment information into it?